include/deal.II/lac/precondition.h

```diff
@@ -856,6 +856,12 @@ class PreconditionPSOR : public PreconditionRelaxation<MatrixType>
  * be used to determine the number of iteration, see e.g. section 5.1 of
  * R. S. Varga, Matrix iterative analysis, 2nd ed., Springer, 2009.
  *
+ * In order to use Chebyshev as a solver, set the degree to
+ * numbers::invalid_unsigned_int to force the automatic computation of the
+ * number of iterations needed to reach a given target tolerance. In this
+ * case, the target tolerance is read from the variable smoothing_range (it
```

Comment:
If you qualify the variable name with the class name, doxygen will automatically link to it.